arch/arm64/boot/dts/qcom/sdm450-samsung-gta2xl-r5.dts

```diff
@@ -0,0 +1,405 @@
+				 <&gcc GCC_MDSS_PCLK0_CLK>,
+				 <&gcc GCC_MDSS_ESC0_CLK>;
+		};
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		volume-up {
+			label = "Volume Up";
+			gpios = <&tlmm 85 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_VOLUMEUP>;
+		};
+	};
+
+	reserved-memory {
+		memory@82800000 {
+			reg = <0x0 0x82800000 0x0 0x2800000>;
+			no-map;
+		};
+
+		memory@85b00000 {
+			reg = <0x0 0x85b00000 0x0 0x600000>;
+			no-map;
+		};
+
+		memory@90001000 {
+			reg = <0x0 0x90001000 0x0 (1080 * 2220 * 3)>;
+			no-map;
+		};
+
+		memory@86100000 {
+			compatible = "ramoops";
+			reg = <0 0x86100000 0 0x100000>;
+
+			console-size = <0x40000>;
+			ftrace-size = <0x40000>;
+			pmsg-size = <0x40000>;
+			record-size = <0x40000>;
+		};
+	};
+
+	i2c_gpio_1: i2c-gpio-1 {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 16 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 17 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+
+	i2c_gpio_2: i2c-gpio-2 {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 21 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 20 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+
+	i2c_gpio_3: i2c-gpio-3 {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 39 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 38 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+
+	i2c_gpio_4: i2c-gpio-4 {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 64 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 65 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+
+	i2c_gpio_5: i2c-gpio-5 {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 131 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 128 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+
+	ts_avdd: vreg-ts-avdd {
+		compatible = "regulator-fixed";
+
+		enable-active-high;
+		gpio = <&tlmm 139 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+		regulator-max-microvolt = <3300000>;
+		regulator-min-microvolt = <3300000>;
+		regulator-name = "ts_avdd";
+	};
+
+	ts_vdd: vreg-ts-vdd {
+		compatible = "regulator-fixed";
+
+		enable-active-high;
+		gpio = <&tlmm 139 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+		regulator-max-microvolt = <1800000>;
+		regulator-min-microvolt = <1800000>;
+		regulator-name = "ts_vdd";
+		vin-supply = <&pm8953_s4>;
+	};
+
+	sensors_vdd: vreg-sensors-1v8 {
+		compatible = "regulator-fixed";
+
+		enable-active-high;
+		gpio = <&expander 6 GPIO_ACTIVE_HIGH>;
+		regulator-max-microvolt = <1800000>;
+		regulator-min-microvolt = <1800000>;
+		regulator-name = "sensors_vdd";
+	};
+
+};
+
+&mdss {
+    status = "okay";
+};
+
+&mdss_dsi0 {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&mdss_te_default>;
+
+	vdda-supply = <&pm8953_s3>;
+	vddio-supply = <&pm8953_l6>;
+
+	panel@0 {
+		compatible = "mdss,hx8279-tv101wum-wuxga";
```

Comment:
I think you will use different compatible in panel driver generator.

Reply:
can i leave that empty then?